flink-streaming-java/pom.xml

```diff
@@ -52,7 +52,7 @@ under the License.
 
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-clients_${scala.binary.version}</artifactId>
+			<artifactId>flink-optimizer_${scala.binary.version}</artifactId>
```

Comment:
can we remove the `flink-optimizer` dependency? Maybe we have to move `FlinkPlan` and `StreamingPlan` out of `flink-optimizer` for that.